prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -0,0 +1,127 @@
+        suppressHl7Fields: "PID-39-1, PID-39-2, PID-39-3"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "ONLY_DIGITS_IN_COMPONENT_ONE"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "ORGANIZATION_NAME"
+        valueSetOverrides: {}
+        nameFormat: "washington"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: true
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: true
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'WA') or (Bundle.entry.resource.ofType(Patient).address.state = 'WA')"
+      qualityFilter:
+              # "doesNotMatch(patient_city,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city = 'UNKNOWN')"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city = 'unknown')"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city = 'Unknown')"
+              # "doesNotMatch(patient_first_name,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).name.given = 'UNKNOWN'"
+              - "Bundle.entry.resource.ofType(Patient).name.given = 'Unknown'"
+              - "Bundle.entry.resource.ofType(Patient).name.given = 'unknown'"
+              # "doesNotMatch(patient_last_name,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).name.family = UNKNOWN"
+              - "Bundle.entry.resource.ofType(Patient).name.family = Unknown"
+              - "Bundle.entry.resource.ofType(Patient).name.family = unknown"
+              # "doesNotMatch(patient_state,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state = 'UNKNOWN')"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state = 'unknown')"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state = 'Unknown')"
+              # "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+              - "Bundle.entry.resource.ofType(Patient).address.line.empty() and Bundle.entry.resource.ofType(Patient).address.postalCode.empty() and Bundle.entry.resource.ofType(Patient).telecom.empty()"
```

Comment:
this should be "(Bundle.entry.resource.ofType(Patient).address.line.empty() and Bundle.entry.resource.ofType(Patient).address.postalCode.empty() and Bundle.entry.resource.ofType(Patient).telecom.empty()).count() = 0" or some similar logic. Currently this will only pass if those things ARE empty.

Reply:
Could also be "(Bundle.entry.resource.ofType(Patient).address.line.empty() and Bundle.entry.resource.ofType(Patient).address.postalCode.empty() and Bundle.entry.resource.ofType(Patient).telecom.empty().not())" - I think but I'd have to test that one